test/WebJobs.Script.Tests/SystemTraceWriterTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests
+{
+    public class SystemTraceWriterTests
+    {
+        private readonly SystemTraceWriter _traceWriter;
+        private readonly Mock<ISystemEventGenerator> _mockEventGenerator;
+        private readonly string _websiteName;
+        private readonly string _subscriptionId;
+
+        public SystemTraceWriterTests()
+        {
+            _subscriptionId = "e3235165-1600-4819-85f0-2ab362e909e4";
+            Environment.SetEnvironmentVariable("WEBSITE_OWNER_NAME", $"{_subscriptionId}+westuswebspace");
```

Comment:
Make this class `IDisposable` and reset the environment variables (remove them) when disposed?